File: Python/8py 2/number_to_string.py
"""
|--------------------------------------------------------------------------
| Convert a Number to a String!
|--------------------------------------------------------------------------
|
"""

def number_to_string(num):
   return str(num)

   
number_to_string = lambda n: str(n)


def number_to_string(num):
   return "{}".format(num)

   
def number_to_string(num):
   return "%s" % num

   
def number_to_string(num):
   return ascii(num)

   
def number_to_string(num):
   return f"{num}"

   
def number_to_string(num):
   return num.__str__()

   
print(number_to_string(67), '67')
print(number_to_string(79585), '79585')
print(number_to_string(79585), "79585")
print(number_to_string(1+2), '3')
print(number_to_string(1-2), '-1')

File: PHP/8php 2/numberToString.php
<?php
/*
|--------------------------------------------------------------------------
| Convert a Number to a String!
|--------------------------------------------------------------------------
|
*/

function numberToString($num)
{
   return strval($num);
}


function numberToString( string $n ) : string {
   return (string) $n;
}


function numberToString($num)
{
   return "$num";
}


print_r(numberToString(67), ' 67');